cmd/exporter/exporter.go

```diff
@@ -223,6 +199,7 @@ func main() {
 		klog.Fatalf("%s", fmt.Sprintf("failed to listen and serve: %v", err))
 	case <-signalChan:
 		klog.Infof("Received shutdown signal")
+		accelerator.Shutdown(accelerator.GPU)
```

Comment:
Could you please explain why `Shutdown` requires a type?

Reply:
Yeah this should be changed. I will change it